### ISSUE TITLE
[Documentation] Add tutorial on how to apply MixUp including part of adding to the loss function.

### ISSUE THREAD

**Issue by ternaus** [albumentations-team/albumentations#1604]
We need a tutorial on how to apply MixUp to the data and the loss function.

We probably can do it as a part of the training classification pipeline in Pytorch Lightning.
